make_html.py

```diff
@@ -17,6 +19,17 @@
 
 from htmlify_comment import htmlify_comment
 
+
+github_token = open(os.environ["GITHUB_TOKEN_FILE"]).read().strip(
+) if "GITHUB_TOKEN_FILE" in os.environ else None
```

Comment:
Sorry, my suggestion is that this should actually be
```suggestion
github_token = os.environ.get("GITHUB_TOKEN")
```
as that's easier to configure with github actions

Reply:
will adapt readme as well...